### ISSUE TITLE
Pantalla negra en PSP/PS Vita/PS3

### ISSUE THREAD

**Issue by holictsubasa** [uzuhenry/DW-ESP-retranslation#1]
Al iniciar el juego en estos sistemas, la pantalla se queda en negro después del logo de playstation.

**Comment by infuziondart** [comment]
No funciona en PSP, ni en PS VITA, hice todo lo que pude pero tampoco, no pase de la pantalla negra.

**Comment by MediaMaquina** [comment]
Hola amigos que version usaron de la japonesa 1.0 o 1.1? No lo especifica el readme cual usar.

**Comment by MediaMaquina** [comment]
Hay algo mal con el parcheador. 
No esta haciendo ECC regen, pero incluso con eso el juego no anda.

**Comment by uzuhenry** [comment]
La nueva actualización del parche resuelve problemas con el check anti-piratería del juego. Espero que la nueva actualización funcione en esos sistemas, dejaré el ticket abierto y comunicaré cuando actualice para que podáis responder si esta nueva versión funciona correctamente en cualquier plataforma.

**Comment by MediaMaquina** [comment]
Hola quiero probar pero de donde se descarga la actualizacion? los links dicen 1.00 22/12

**Comment by Jrublgo** [comment]
Yo he entendido que lo comentara por aqui cuando este lista la nueva version. 

Aporto mi granito de arena.

He convertido digimon world a eboot con compresion level 2 con PSX2PSP 1.4.2  para adrenaline via y, tras saltar la tipica escena del logo psx1 salta el error C1-2858-3 Deberia probar con mas configuraciones a ver si es alguna cosa del adrenaline. 

He instalado el retroarch para usar el rearmed (sin BIOS) y me corre el juego (tambien he hecho la prueba en switch, mismo procedimiento y corre minimo hasta que puedes andar libremente con tu digimon) convirtiendo la iso en bin (el error unico, pero ya es dicho, es el tema del sonido)

En unas semanas me llegara otra copia de digimon world 1, por lo que hare nuevamente pruebas para ver si es de la iso que tengo esta dañada o algo

Dejo fotos a modo de evidencia (dato curioso, noto que corre mas rapido en la psvita)
https://imgur.com/a/eE1seGf

Si hay que probar en alguna consola mas (ps2,old3ds,new3ds,wii) puedo ayudar

Saludos.

**Comment by uzuhenry** [comment]
El parche 1.1 ya está publicado, agradecería si pudieseis ayudarnos a probar las compatibilidades de este nuevo parche. El ticket se quedará abierto para que todo el mundo pueda aportar feedback al respecto.

Muchas gracias por los reportes, ¡y esperamos que ahora funcione en todos esos sitemas!

**Comment by MediaMaquina** [comment]
Me funciono bien en la PSP e. 1.1 gracias.

**Comment by holictsubasa** [comment]
Confirmo que la versión 1.1 funciona en PS3 con CFW Evilnat, en formato Bin. También funciona en ps vita desde adrenaline y desde retroarch, convirtiéndolo en formato EBOOT.PBP

**Comment by Jrublgo** [comment]
Yo tambien confirmo que funciona en PSvita con retroarch rearmed y desde adrenaline. En ordenador he probado con el emulador beetle hw y lo unico que he visto es que los videos no me carga y debo skipearlos (me alerta de bad sector)  Tambien noto que las pantallas de carga van mas lentas que en otros emuladores. Ire editando conforme vaya probando en mas plataformas

**Comment by uzuhenry** [comment]
> Yo tambien confirmo que funciona en PSvita con retroarch rearmed y desde adrenaline. En ordenador he probado con el emulador beetle hw y lo unico que he visto es que los videos no me carga y debo skipearlos (me alerta de bad sector) Tambien noto que las pantallas de carga van mas lentas que en otros emuladores. Ire editando conforme vaya probando en mas plataformas

https://www.romhacking.net/utilities/1264/

puedes confirmar si con esto se te arreglan las cinemáticas?

**Comment by Jrublgo** [comment]
@uzuhenry Yep, gracias! Efectivamente arregla el bug con las cinematicas de ese emulador!

**Comment by Jrublgo** [comment]
Buenas, pues he detectado otro error en el emulador de adrenaline para psvita. Por lo visto skipea algunos eventos (a mi me ha pasado con el dialogo de insekimon, no se si habrá más) No he encontrado solucion alguna, asi que he porteado el save con MemCard Rex y luego he importado el save de MC al duckduckstation y alli si salta. El error se indentifica facilmente porque no aparece la accion que debia aparecer sino que puedes entrar en la batalla con el digimon de manera infinitas.

**Comment by uzuhenry** [comment]
Debido a una instrucción "especialita" insekimon en sí daba muchos errores y cuelgues, por desgracia aleatorios. En la futura versión del parche he intentado corregirlo cambiando la instrucción y parece funcionar mejor (aunque se para la bgm por el fix, pero mejor eso a un softlock). Cuando publique la nueva versión prueba de nuevo a ver si te funciona bien.

**Comment by oriolmollet** [comment]
pequeño problema, no puedo reclutar a betamon, cuando itento ir a la zona del manglar donde esta, el juego se cuelga. debe de ser por el parche de continuacion de musica o algo.

**Comment by uzuhenry** [comment]
> pequeño problema, no puedo reclutar a betamon, cuando itento ir a la zona del manglar donde esta, el juego se cuelga. debe de ser por el parche de continuacion de musica o algo.

Buenas, acabo de probarlo con el último parche y el anterior y funciona perfectamente en una iso parcheada simple.
Es posibles que tengas activados el randomizador de enemigos? debido a la imposibilidad de poder testear todas las combinaciones del randomizer hay algunas incompatibles entre sí. Si randomizaste enemigos o algún otro elemento, podrías pasarme el spoiler log? Así puedo revisar a fondo qué es lo que ocurre.

Como solución puedes parchear una nueva iso co el randomizador de enemigos, ya que es muy difícil que el randomizer repita ese emparejamiento problemático.

**Comment by oriolmollet** [comment]
[DigimonWorld (Japan).iso-Parcheado.iso_spoilerlog.txt](https://github.com/uzuhenry/DW-ESP-retranslation/files/13718654/DigimonWorld.Japan.iso-Parcheado.iso_spoilerlog.txt)


> > pequeño problema, no puedo reclutar a betamon, cuando itento ir a la zona del manglar donde esta, el juego se cuelga. debe de ser por el parche de continuacion de musica o algo.
> 
> Buenas, acabo de probarlo con el último parche y el anterior y funciona perfectamente en una iso parcheada simple. Es posibles que tengas activados el randomizador de enemigos? debido a la imposibilidad de poder testear todas las combinaciones del randomizer hay algunas incompatibles entre sí. Si randomizaste enemigos o algún otro elemento, podrías pasarme el spoiler log? Así puedo revisar a fondo qué es lo que ocurre.
> 
> Como solución puedes parchear una nueva iso co el randomizador de enemigos, ya que es muy difícil que el randomizer repita ese emparejamiento problemático.

[DigimonWorld (Japan).iso-Parcheado.iso_spoilerlog.txt](https://github.com/uzuhenry/DW-ESP-retranslation/files/13718662/DigimonWorld.Japan.iso-Parcheado.iso_spoilerlog.txt)

**Comment by uzuhenry** [comment]
Tras comprobarlo, efectivamente el par randomizado de esa zona es problemático. PAra la siguient actualización añadiré otra excepción al randomizer para evitar ese emparejamiento.
